app/views/layouts/application.html.erb

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Ruby &amp; Rosé</title>
-  <%= favicon_link_tag 'logo.png' %>
+  <%= favicon_link_tag 'zap-logo.png' %>
```

Comment:
yo Alan do we know if all browsers can support a favicon with this png size/file type? Or do we need a `.ico` file type as well?